### ISSUE TITLE
RTCVideoView might have a bug

### ISSUE THREAD

**Issue by linguitang** [cloudwebrtc/flutter-webrtc#54]
between send offer success and unreceived target answer,I show _localRander full screen
```
new Stack(children: <Widget>[
  new Positioned(
	  left: 0.0,
	  right: 0.0,
	  top: 0.0,
	  bottom: 0.0,
	  child: new Container(
		margin: new EdgeInsets.fromLTRB(0.0, 0.0, 0.0, 0.0),
		width: MediaQuery.of(context).size.width,
		height: MediaQuery.of(context).size.height,
		child: new RTCVideoView(_localRenderer),
		decoration: new BoxDecoration(color: Colors.black54),
	  )),
	])
```
it work well.

when I receive target answer, I want show _remoteRenderer full screen.
```
new Stack(children: <Widget>[
                  new Positioned(
                      left: 0.0,
                      right: 0.0,
                      top: 0.0,
                      bottom: 0.0,
                      child: new Container(
                        margin: new EdgeInsets.fromLTRB(0.0, 0.0, 0.0, 0.0),
                        width: MediaQuery.of(context).size.width,
                        height: MediaQuery.of(context).size.height,
                        child: new RTCVideoView(_remoteRenderer),
                        decoration: new BoxDecoration(color: Colors.black54),
                      )),])
```
it work fail. it also show _localRenderer full screen.
but when I change code like this:
```
new Stack(children: <Widget>[
          new Positioned(
                      left: 0.0,
                      right: 0.0,
                      top: 0.0,
                      bottom: 0.0,
                      child: new Container(
                        margin: new EdgeInsets.fromLTRB(0.0, 0.0, 0.0, 0.0),
                        width: MediaQuery.of(context).size.width,
                        height: MediaQuery.of(context).size.height,
                        child: new RTCVideoView(_remoteRenderer),
                        decoration: new BoxDecoration(color: Colors.black54),
                      )),
                  new Positioned(
                      left: 0.0,
                      right: 0.0,
                      top: 0.0,
                      bottom: 0.0,
                      child: new Container(
                        margin: new EdgeInsets.fromLTRB(0.0, 0.0, 0.0, 0.0),
                        width: MediaQuery.of(context).size.width,
                        height: MediaQuery.of(context).size.height,
                        child: new RTCVideoView(_remoteRenderer),
                        decoration: new BoxDecoration(color: Colors.black54),
                      )),])
```

just add Positioned Widget again.it work well,show _remoteRenderer now.

**Comment by linguitang** [comment]
Q 2： when I made p2p call such as demo example, how can I just play target sound?now we can listen two randers sound

**Comment by linguitang** [comment]
Q3: How can I switch camera？ Because phone has two camera

**Comment by cloudwebrtc** [comment]
> Q3: How can I switch camera？ Because phone has two camera

Please use MediaStreamTrack.switchCamera.

**Comment by linguitang** [comment]
> > Q3: How can I switch camera？ Because phone has two camera
> 
> Please use MediaStreamTrack.switchCamera.

I try this:
```
    List<MediaStreamTrack> tracks = _localStream.getVideoTracks();
    for (var i = 0; i < tracks.length; ++i) {
      tracks[i].switchCamera();
    }
```

No effect

**Comment by cloudwebrtc** [comment]
> > > Q3: How can I switch camera？ Because phone has two camera
> > 
> > 
> > Please use MediaStreamTrack.switchCamera.
> 
> I try this:
> 
> ```
>     List<MediaStreamTrack> tracks = _localStream.getVideoTracks();
>     for (var i = 0; i < tracks.length; ++i) {
>       tracks[i].switchCamera();
>     }
> ```
> No effect

I tested it, this is indeed a plugin issue, and it has been fixed, please update to version 0.1.0

**Comment by linguitang** [comment]
> > > > Q3: How can I switch camera？ Because phone has two camera
> > > 
> > > 
> > > Please use MediaStreamTrack.switchCamera.
> > 
> > 
> > I try this:
> > ```
> >     List<MediaStreamTrack> tracks = _localStream.getVideoTracks();
> >     for (var i = 0; i < tracks.length; ++i) {
> >       tracks[i].switchCamera();
> >     }
> > ```
> > No effect
> 
> I tested it, this is indeed a plugin issue, and it has been fixed, please update to version 0.1.0

I has updated.it work well.

How about the other two questions?

**Comment by cloudwebrtc** [comment]
> Q 2： when I made p2p call such as demo example, how can I just play target sound?now we can listen two randers sound

You can find the track and use ```track. enabled = false``` to mute it.

and about Q 1.

The rendering problem may be related to the flutter layout. Usually the rendering is controlled by flutter after the video stream is sent to the texture. If the screen cannot appear, you can try different layouts, especially when one control is covered by another.
You can provide some screenshots for better analysis.
